CHANGELOG.rst

```diff
@@ -6,24 +6,38 @@ Rasa Change Log
 All notable changes to this project will be documented in this file.
 This project adheres to `Semantic Versioning`_ starting with version 1.0.
 
-[Unreleased 1.1.6] - `master`_
+[Unreleased 1.1.7] - `master`_
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 Added
 -----
-- rest channel supports setting a message's input_channel through a field
-  ``input_channel`` in the request body
+
 
 Changed
 -------
-- recommended syntax for empty ``use_entities`` and ``ignore_entities`` in the domain file
-  has been updated from ``False`` or ``None`` to an empty list (``[]``)
 
 
 Removed
 -------
 
 
+Fixed
+-----
+
```

Comment:
should I remove the empty ones?

Reply:
I removed the empties from 1.1.6, I thought we keep them for the new master version

Reply:
omg, its late

Reply:
Don't stay to late 😘